document.go

```diff
@@ -76,3 +76,12 @@ type GeoPointField interface {
 	Lon() (float64, error)
 	Lat() (float64, error)
 }
+
+// TokenisableSpatialField is an optional interface for fields that
+// supports pluggable custom hierarchial spatial token generation.
+type TokenisableSpatialField interface {
```

Comment:
Should this be Tokenizable.. ?

Reply:
My next commit is touching this file too. So, Let me fix it then.